src/main/java/bio/terra/janitor/common/exception/DuplicateLabelException.java

```diff
@@ -0,0 +1,12 @@
+package bio.terra.janitor.common.exception;
```

Comment:
This class are specific for janitor services, it only happens when you get key conflict when inserting data into our Label table. But the patten are the same(the supre calss - BadRequestException, ErrorReport exception). This is the pattern in WM and DB, for all errors they create different exceptions classes. If we (and I think) we use the same schema for error_report in response, many classes can be in common classes, and we extend them like this class. I had a TODO in GlobalExceptionHandler saying use common library once we have

Reply:
Sorry, I was asking if these exceptions that are specific to the JanitorService/JanitorDao class should go in the packages that throw them instead of in the janitor.common.exception package. For example, data repo defines some exceptions for its JobService under that package even though those exceptions extend common exception types. https://github.com/DataBiosphere/jade-data-repo/tree/develop/src/main/java/bio/terra/service/job/exception

Reply:
I am concerned that those exception would be everwhere. I feel it is better to have a central package to all exceptions.
Not sure which way is better, WM did the janitor pattern I think.

Reply:
sgtm. I asked Dan, about these and he said "I think the theory is that it is hard to predict over the long term which exceptions you will want to catch, so by using specific exceptions you keep your options open. But it is kind of like coding conventions where everybody has their own opinion and there isn’t one obvious right answer. I find it a little more helpful to have specific ones because the ide can help track them down more easily than a string search."
So my personal take would be to err towards less exceptions types until they're needed since we're not publishing this code as a library and can change it later without regard for backwards compatibility.